ethree-common/src/main/java/com/virgilsecurity/android/common/storage/local/FileGroupStorage.kt

```diff
@@ -111,6 +112,16 @@ internal class FileGroupStorage internal constructor(
         fileSystemEncrypted.write(data, GROUP_INFO_NAME, subdir)
     }
 
+    internal fun setParticipants(newParticipants: Set<String>, sessionId: Data) {
+        val tickets = retrieveLastTickets(1, sessionId)
```

Comment:
Should we verify that newParticipants set is not empty?

Reply:
It's been checked here: https://github.com/VirgilSecurity/virgil-e3kit-kotlin/blob/0a6a031fde4f2ba7e8ff2ba25b58d7839fb7bf71/ethree-common/src/main/java/com/virgilsecurity/android/common/model/Group.kt#L260